.eslintrc.js

```diff
@@ -25,6 +31,7 @@ module.exports = {
     'react-hooks',
     'prettier',
     '@typescript-eslint',
+    'import',
```

Comment:
Is this make some real changes during coding?

Reply:
This plugin *was* used also before, just hidden inside `airbnb` config. So I dropped `airbnb` and configured it as standalone. Now it seems to work better, for example is has spotted unused import in `UserSettingsScreen.tsx` component, you can check it in diff.